packages/ssx-sdk/tests/modules/Encryption.test.ts

```diff
@@ -1,35 +1,58 @@
+import { ethers } from 'ethers';
 import { generateTestingUtils } from 'eth-testing';
 import { TextEncoder as TE, TextDecoder as TD } from 'util';
 global.TextEncoder = TE;
 global.TextDecoder = TD;
 
+// import crypto from "@trust/webcrypto";
+// Object.defineProperty(global.self, "crypto", {
+//   ...crypto,
+//   value: {
+//     ...crypto.subtle,
+//     subtle: {
+//       importKey: () => new CryptoKey()
+//     },
+//   },
+// });
```

Comment:
@skgbafa this is what I tried to mock the crypto.subtle.importKey and avoid the error.

Reply:
I spent quite a bit of time on this test as well. I'll have to look at the JOSE library to see why the jest tests are failing. I think because we are merging this into the `ssx-modules` branch and not `main`, we can focus on making sure all the tests are passing when merging into `main`. The functionality works, but not the jest tests

Reply:
Agree!

Reply:
same